docs/development_suite/webhooks-and-events/events.mdx

```diff
@@ -25,57 +25,466 @@ This event will be triggered when a project is created successfully.
 
 #### Event `payload`
 
-| Property    | Value    | Description                                          |
-|-------------|----------|------------------------------------------------------|
-| `tenantId`  | `string` | The identifier of the Company the Project belongs to |
-| `projectId` | `string` | The identifier of the newly created Project          |
+| Property    | Value    | Description                                                |
+|-------------|----------|------------------------------------------------------------|
+| `tenantId`  | `string` | The identifier of the Company the Project belongs to       |
+| `projectId` | `string` | The identifier of the newly created Project                |
+| `project `  | `object` | The project metadata (ideally, the whole project resource) |
 
 <Tabs>
 <TabItem value="example" label="Example" default>
 
 ```json
 {
-    "eventName": "project_created",
-    "eventTimestamp": 123456,
+  "eventName": "project_created",
+  "eventTimestamp": 123456,
+  "payload": {
+    "tenantId": "my-company-id",
+    "projectId": "0000000000000001"
+  }
+}
+```
+
+</TabItem>
+<TabItem value="json-schema" label="JSON Schema">
+
+```json
+{
+  "additionalProperties": false,
+  "properties": {
+    "eventName": {
+      "const": "project_created",
+      "type": "string"
+    },
+    "eventTimestamp": {
+      "type": "number"
+    },
     "payload": {
-      "tenantId": "my-company-id",
-      "projectId": "0000000000000001"
+      "additionalProperties": false,
+      "properties": {
+        "projectId": {
+          "type": "string",
+          "description": "The identifier of the Project that was created"
+        },
+        "tenantId": {
+          "type": "string",
+          "description": "The identifier of the Company the Project belongs to"
+        },
+        "project": {
+          "type": "object",
+          "description": "The project metadata (ideally, the whole project resource)"
```

Comment:
We should use the $ref

Reply:
Why? Users will see like in the attached picture
![Schermata 2023-07-03 alle 11 48 35](https://github.com/mia-platform/documentation/assets/22480239/2ccfd6d0-828f-401a-ab6b-24da2ff9b05f)